packages/haring-react/src/Components/FloatingMenu/FloatingMenu.tsx

```diff
@@ -0,0 +1,131 @@
+'use client';
+import type { ReactElement } from 'react';
+
+import { ActionIcon, Flex, Group, Modal, Text } from '@mantine/core';
+import { useMediaQuery } from '@mantine/hooks';
+import {
+  CalendarBlank,
+  FileText,
+  Newspaper,
+  Question,
+  Star,
+} from '@phosphor-icons/react';
+import { useState } from 'react';
+
+import classes from './FloatingMenu.module.css';
+
+type IMenuPosition = 'Left' | 'Right';
+
+export interface IMenuItems {
+  hasModal: boolean;
+  href: string;
+  id: number;
+  text: string;
+}
+
+export interface IFloatingMenuProps {
+  items?: IMenuItems[];
+  position?: IMenuPosition;
+}
+
+export function FloatingMenu({
+  position = 'Right',
+  items,
+}: IFloatingMenuProps): ReactElement {
```

Comment:
Pour une meilleur homogénéité du code il faudrait que le destructuring de props soit fait sur une ligne à part.
C'est à dire:
```ts
export function FloatingMenu(props: IFloatingMenuProps): ReactElement {
  const { position = 'right', items } = props;
  // ...
}
```